### ISSUE TITLE
Balance Sheet issue

### ISSUE THREAD

**Issue by codemvp** [codemvp/accountgo#10]
Reported by Romi

romi0199: I was looking into the demo and found problem in the balancesheet. Here Assest !=liabilities+O.E is showing.

romi0199: http://accountgo.apphb.com/Financial/BalanceSheet

**Comment by codemvp** [comment]
Additional details:

Problem: The balance sheet Asset is not equal to Liabilities + Capital.

Root Cause: The net income in "Income Statement" is not assigned to account for the accumulated profit/loss.

Resolution: Assign the net income to accumulated profit/loss.

**Comment by codemvp** [comment]
public ActionResult BalanceSheet()
        {
            var model = _financialService.BalanceSheet();
            var incomestatement = _financialService.IncomeStatement();
            var netincome = incomestatement.Where(a => a.IsExpense == false).Sum(a => a.Amount) - incomestatement.Where(a => a.IsExpense == true).Sum(a => a.Amount);

            // TODO: Add logic to get the correct account for accumulated profit/loss. Currently, the account code is hard-coded here.
            // Solution 1: Add two columns in general ledger setting for the account id of accumulated profit and loss.
            // Solution 2: Add column to Account table to flag if account is net income (profit and loss)
            if (netincome < 0)
            {
                var loss = model.Where(a => a.AccountCode == "30500").FirstOrDefault();
                loss.Amount = netincome;
            }
            else
            {
                var profit = model.Where(a => a.AccountCode == "30400").FirstOrDefault();
                profit.Amount = netincome;
            }

            return View(model);
        }
